topostats/default_config.yaml

```diff
@@ -13,7 +13,7 @@ filter:
   threshold_std_dev: 1.0
   threshold_absolute_lower: -1.0
   threshold_absolute_upper: 1.0
-  gaussian_size: 0.5
+  gaussian_size: 1.0121397464510862 # in px
```

Comment:
I'm naive (or ignorant!), what is the reason for setting the `gaussian_size` to this value?

Could we perhaps document why this has been chosen as a default value somewhere? If not in the formal `docs/` then a comment here in the `default_config.yaml` so that when we get round to writing up more documentation we have a record of why its chosen.

Better still an explanation of what `gaussian_size` is actually doing would be useful, as I say I don't know what it means.

Reply:
This value is used as it is the same value as before, just in different units. i.e. a `gaussian_size` of 1.0121397464510862 px == 0.5 nm. This value means that the tests do not need to be changed, but it could just as easily assume any other value, but the tests would need to be modified.

I'll add the below to the `docs/configuration.md` file:
"The number of standard deviations to build the Gaussian kernel and thus affects the degree of blurring. See [skimage.filters.gaussian](https://scikit-image.org/docs/dev/api/skimage.filters.html#skimage.filters.gaussian) and `sigma` for more information."

Does that description cover it?

Reply:
Great thanks. :+1: 

Will this vary depending on the resolution/size of the scan and/or instrument and/or the `pixel_to_nm_scaling` that is extracted from the scans?

Reply:
Nope. So the idea is that this is now set across all the images using a single value which does not change between images and thus doesn't use the `pixel_to_nm_scaling` or anything else.